src/nanoemoji/svg.py

```diff
@@ -405,14 +415,15 @@ def _picosvg_docs(
         reuse_cache = ReuseCache()
         # establish base svg, defs
         svg = SVG.fromstring(
-            r'<svg version="1.1" xmlns="http://www.w3.org/2000/svg"><defs/></svg>'
-        )
+            r'<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs/></svg>'
+        , False)
 
         for color_glyph in (color_glyphs[g] for g in group):
             _add_glyph(svg, color_glyph, reuse_cache)
 
         gids = tuple(color_glyphs[g].glyph_id for g in group)
-        doc_list.append((svg.tostring(), min(gids), max(gids)))
+        doc_list.append((svg.tostring(False), min(gids), max(gids)))
```

Comment:
we don't need to have an option to keep xmlns, that's added or removed automatically depending on whether any xlink attribute is actually used. We can close https://github.com/googlefonts/nanoemoji/pull/270

```suggestion
        doc_list.append((svg.tostring(), min(gids), max(gids)))
```